rfc/0008-upgrade-statement.md

```diff
@@ -0,0 +1,103 @@
+|              |                                  |
+| :----------- | :------------------------------- |
+| Feature Name | Upgrade Statement                |
+| Start Date   | 05/23/2022                       |
+| Category     | updates and versioning           |
+| RFC PR       | https://github.com/kubewarden/rfc/pull/6  |
+| State        | ~**ACCEPTED**~                   |
+
+
+# Summary
+[summary]: #summary
+
+This RFC proposes how the upgrade of the Kuberwarden stack will work and how to do it.
+
+# Motivation
+[motivation]: #motivation
+
+Kubewarden is reaching the v1.0 milestone and we need to define a upgrade process
+to users and for the development team to follow. Therefore, users will are be able
+to plan when and how to upgrade their Kuberwarden stack. And the Kuberwarden developers
+will be able to define when and how ship new features and fixes.
+
+## Examples / User Stories
+[examples]: #examples
+
+As Kubewarden user, I want to know, among all Kubewarden components, what are the
+compatible versions.
+
+As Kubewarden user, I want to know the upgrade path of the Kuberwarden components.
+
+As Kubewarden developer, I want to know which components I need to upgrade in order
+to release a feature of a bug fix.
+
+# Detailed design
+[design]: #detailed-design
+
+This document describes how Kubewarden versioning works and how users can upgrade
+their stack. The Kubewarden project uses [Semantic versioning](https://semver.org/)
+to define the version of all its components. In other words, the version follows
+the `MAJOR.MINOR.PATCH` pattern.  The supported versions are the two latest releases.
+This means that if the latest version released of the Kubewarden components is
+`1.2.0`. Both `1.2.0` and `1.1.0` version are under active maintenance.
```

Comment:
I would keep just the latest release under active maintenance

Reply:
me too, to be fair. I don't want to backport fixes to previous releases.